playbooks/10 - SP - Time Series Charts/{POST UPDATE] Trigger Updated Chart Queries.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
name of this PB needs correction. Please follow standard guidelines or look for SFSP PB names as a reference

Reply:
I've updated the name of the post create and post update playbooks to more closely match those in SFSP